src/screens/App/components/Navbar/components/UserPopover/index.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+import { withRouter } from 'react-router-dom'
+
+import UserPopover from './layout';
+
+class UserPopoverContainer extends Component {
+
+  logout = () => {
```

Comment:
as a convention, we use `handle` in this cases to reference method that handle the action and `on` who receive and execute this handler.
In this case.
```
handleLogout = () => {
.....
}

render() {
       return <UserPopover onLogout={this.logout}/>
   }

```